fix: make infra state more polished and add doc (#50)
graasp/graasp-infrastructure: README.md

export AWS_ACCESS_KEY_ID="yourkey"
export AWS_SECRET_ACCESS_KEY="yoursecret"

> # used to defined the state of the deployment

Reviewer: "used to **define**..."

In this case the fix is usually to update the `@cdktf/provider-aws` dependency. Using `yarn upgrade-interactive` it is easy to do.

> If upgrading the npm dependencies does not solve the problem, it might be because the required ranges are incompatible. This can be cause by modules requiring higher ranges, and the pre-built aws provider not being updated to those ranges yet. In this case, look at the requirements for the two modules that are manually added in the `cdktf.json` config file and set them to a previous version that makes the range requirement from the pre-built aws provider happy.

Reviewer 2: ```suggestion
If upgrading the npm dependencies does not solve the problem, it might be because the required ranges are incompatible. Some modules might require higher ranges, and the pre-built aws provider not being updated to those ranges yet. In this case, look at the requirements for the two modules that are manually added in the `cdktf.json` config file and set them to a previous version that makes the range requirement from the pre-built aws provider happy.
```
